### ISSUE TITLE
Responsive view does not ask for email address or recognize custom product name

### ISSUE THREAD

**Issue by futuresoup** [mozilla/persona#3974]
New users to Thimble on http://thimble.webmaker.org are getting confused because in some cases, hitting "sign in" button triggers the smaller responsive view of the Persona window.

This responsive view does not ask the user to enter their email address, and instead of saying "sign in to webmaker" it says "sign in with persona" - which is also confusing.

Can we add text that asks first time users to enter their email address in order to get signed up for Webmaker? And can the responsive view recognize the product name better (Webmaker in this case) like it does for the normal size view.

Video of the issue here - https://www.dropbox.com/s/gwkfa7lj24o87nf/thimble%20persona%20bug.mov

More context on Bugzilla here - https://bugzilla.mozilla.org/show_bug.cgi?id=921559

Thanks!

**Comment by shane-tomlinson** [comment]
Thanks for the feedback @futuresoup - This was fixed last week with #3957 - more specifically, this change: https://github.com/mozilla/browserid/pull/3957/files#diff-278fdfa06a730b614a46b9241eace5e3R25. This fix should roll into production in about 3.5 week's time.

Closing as fixed.

**Comment by shane-tomlinson** [comment]
@futuresoup - The part about not asking for the email address was an intentional design decision made by @mhanratty and @skinny97214 whenever designing the screens for mobile users.

**Comment by futuresoup** [comment]
Hi Shane - can you please clarify which issue was fixed?

Ideally it would be the issue where the Persona registration window shows up in a normal sized view when signing in from Thimble instead of the responsive view. Some folks can reproduce this issue and some can't. So far, the only common factor for reproduction has been 13" Macbooks or Macbook pro screens.

If not asking for the email address is going to remain in the design, then I think we will still run into the problem where first time users land into Thimble, start making something, decide to save, and then don't understand how to register because they hit the responsive view that doesn't ask for an email address.

Thanks for your attentive comments so far :)

**Comment by shane-tomlinson** [comment]
@futuresoup - we fixed this part:

> instead of saying "sign in to webmaker" it says "sign in with persona" - which is also confusing.

Users who see the mobile design _usually_ have zoomed out on the page they are trying to sign in to. You can try this yourself by visiting https://thimble.webmaker.org/, zoom out one level from the default, and open the Persona dialog. This is because the browser transfers zoom settings from a parent to child window (see #2008 and https://bugzilla.mozilla.org/show_bug.cgi?id=779335). 

Unfortunately, without a ton of work, there isn't a ton we can do to resolve the zoom issue. If you have access to any of the users who see the mobile view, can you see if they have modified the zoom settings on Thimble? If they have and reset the zoom settings, does the Persona window show in desktop mode as expected?

**Comment by futuresoup** [comment]
Ah groovy, just verified that when I reset my zoom settings, I get the normal persona window instead of the responsive one  :)
It is strange though that when I zoom out just one level, the parent window makes everything smaller (zoomed out), but the persona window acts the opposite and makes everything larger (as if zoomed in).

I will trust @mhanratty and @skinny97214's design choice in regards to not asking for the email address instead of trying to argue it. I guess if you're on a phone the experience is different, but I don't quite understand how people will figure out what to do without being prompted for their email address. Ah well glad the branding issue got fixed and I figured out what was causing the original error. 

Thanks!!
